lib/elm-test.js

```diff
@@ -86,20 +83,17 @@ function resolveFilePath(filename) {
   });
 }
 
-var pathToMake = undefined;
-var pathToElmPackage = "elm-package";
+var pathToElm = "elm";
 
 if (args.compiler !== undefined) {
-  pathToMake = path.resolve(args.compiler);
+  pathToElm = path.resolve(args.compiler);
```

Comment:
:art:  This is extremely minor, but `pathToElm` to me could be read as `path to elm files`, rather than `path to elm binary`

Reply:
Renamed to `pathToElmBinary` in 5316b2c